allennlp/tests/semparse/contexts/table_question_context.py

```diff
@@ -0,0 +1,74 @@
+# pylint: disable=no-self-use,invalid-name,protected-access,too-many-public-methods
```

Comment:
This file needs to have `test` in its name, or pytest won't pick it up as containing tests.  That's why CI is saying that your test coverage on the additions here is basically zero.  So I'd rename this to `table_question_context_test.py`.